README.rst

```diff
@@ -74,6 +74,29 @@ US/Canada, a short code for the UK and an alphanumertic originator for everywher
     originators.44 = short:12345
     originators.default = alpha:Matrix
 
+Docker
+======
+
+A Dockerfile is provided for sydent. To use it, run ``docker build -t sydent .`` in a sydent checkout.
+To run it, use ``docker run --env=SYDENT_SERVER_NAME=my-sydent-server --mount type=volume,source=sydent-data,destination=/data sydent``.
```

Comment:
When running the docker container, I'm unable to access Sydent at `localhost:8090`, nor `my-docker-internal-ip-address:8090`. According to the [Dockerfile reference](https://docs.docker.com/engine/reference/builder/#expose), it looks as if `-p 8090:8090` must still be specified when running the container to access the service externally.

It would be good to note that here.

Reply:
I updated the docker example and add more explanaition about persistent data handling.